st2common/st2common/util/actionalias_matching.py

```diff
@@ -95,3 +97,27 @@ def match_command_to_alias(command, aliases):
 
             results.append((alias, format_string[0], format_string[1]))
     return results
+
+
+def get_matching_alias(command):
+    """
+    Find a matching ActionAliasDB object (if any) for the provided command.
+    """
+    # 1. Get aliases
+    action_alias_dbs = ActionAlias.query(enabled=True)
```

Comment:
If it turns out that retrieving all the aliases on each API endpoint is too slow, we can add some logic to cache it in memory inside the API process (controller).